br/cmd/br/stream.go

```diff
@@ -108,6 +109,21 @@ func newSteramResumeComamnd() *cobra.Command {
 	return command
 }
 
+func newStreamStatusComamnd() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "status",
+		Short: "status a stream task",
```

Comment:
```suggestion
		Short: "get status of a stream task",
```

status is not a verb